### DISCUSSION TITLE
F-Droid Release missing

### DISCUSSION THREAD

**Discussion by Antarctris** [Swordfish90/Lemuroid#600] [General]
Hi there,

First of all, great work!
I just wanted to know if and when F-Droid will receive a version update. I am currently using Android 13  with F-Droid, no Play-Store, and would love to get recent bug fixes.
No pressure intended, I'm not in a hurry, just asking.

Kind regards

**Comment by foundObjects** [comment]
> Hi there,
> 
> First of all, great work!
> I just wanted to know if and when F-Droid will receive a version update. I am currently using Android 13  with F-Droid, no Play-Store, and would love to get recent bug fixes.
> No pressure intended, I'm not in a hurry, just asking.
> 
> Kind regards

You might find the following helpful for github based apps that aren't updated regularly on fdroid:

Get an RSS reader like Feeder (fdroid) or Read You (https://github.com/Ashinch/ReadYou) then find the GitHub releases pages for your various apps that aren't on fdroid or aren't updated at fdroid in a timely fashion. Take the releases URL and punch it into your RSS reader with `.atom` on the end, eg: https://github.com/Ashinch/ReadYou/releases.atom, add that as a feed and set the reader to notify you when there's a new entry. You'll be notified for every new release.

I use this to track anything I use on Android that's not a store app. It's also just all around handy for notification on any change to a project you want to watch for, basically any query against a github repo will return an atom feed if you add .atom to the end of it.

**Reply by Antarctris** [reply]
This is a nice suggestion, but unfortunately will not work for this project, as the releases aren't even updated here on github. See the last release here is from summer last year and is also the same release available on F-Droid.

**Comment by Algiuxs** [comment]
Try using obtainium, then pasting in the lemuroid github link, it will automatically update the lemuroid, no play store needed.
Also obtainium is open source.
